### PR TITLE
[UX] Add a new task to the plan (drop-down menu)

### PR DESCRIPTION
**Pull request by kwbak**
fixmov

Need fixes: Overlay does not cover header bar

### CODE REVIEW

File: src/screens/planActivity/FixedCreatePlanSubItemButton.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      actionName: name,
+    });
+  };
+
+  render() {
+    return (
+      <FloatingAction
+        color={palette.secondary}
+        actions={actions}
+        overlayColor="rgba(97, 91, 185, 1)"
```
Comment (maciejhein) [inline src/screens/planActivity/FixedCreatePlanSubItemButton.tsx]: I think that you should move that color to palette or use existing color from palette.

File: src/screens/planActivity/FixedCreatePlanSubItemButton.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+interface State {
+  actionName: string;
+}
+
+export class FixedCreatePlanSubItemButton extends React.PureComponent<{}, State> {
+  state: State = {
+    actionName: '',
+  };
+
+  onPressItem = (name: any) => {
```
Comment (maciejhein) [inline src/screens/planActivity/FixedCreatePlanSubItemButton.tsx]: Maybe you should add type to name parameter

File: src/screens/planActivity/FixedCreatePlanSubItemButton.tsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+const actions = [
+  {
+    name: 'create-task',
+    icon: <Icon name="layers" type="material" color={palette.primary} size={22} />,
+    text: i18n.t('updatePlan:addTask'),
+    position: 1,
+    color: palette.background,
+    textBackground: palette.primaryVariant,
+    textStyle: { fontSize: 12, fontFamily: fonts.sansSerif.medium },
```
Comment (mgolawski) [inline src/screens/planActivity/FixedCreatePlanSubItemButton.tsx]: I think that we should use `typography` here.

File: src/screens/planActivity/FixedCreatePlanSubItemButton.tsx [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+import { Animated, StyleSheet } from 'react-native';
+import { FloatingAction } from 'react-native-floating-action';
+
+import { Icon } from 'components';
+import { i18n } from 'locale';
+import { fonts, palette } from 'styles';
+
+const actions = [
+  {
+    name: 'create-task',
```
Comment (mgolawski) [inline src/screens/planActivity/FixedCreatePlanSubItemButton.tsx]: Store the action names in variables.

File: src/screens/planActivity/FixedCreatePlanSubItemButton.tsx [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+  };
+
+  renderFloatingActionButton = () => {
+    return (
+      <FloatingAction
+        color={this.state.isOpen ? palette.secondary : palette.primaryVariant}
+        actions={actions}
+        showBackground={false}
+        onPressItem={this.onPressItem}
+        floatingIcon={
```
Comment (mgolawski) [inline src/screens/planActivity/FixedCreatePlanSubItemButton.tsx]: I suggest to extract this if to separate function inside component body. Maybe `renderFloatingIcon()`

**Pull request merged**